src/Content/Item.php

```diff
@@ -796,4 +733,24 @@ private function getShareArray(array $item): array
 
 		return [];
 	}
+
+	/**
+	 * Add a link to a shared post at the end of the post 
+	 *
+	 * @param string  $body
+	 * @param integer $quote_uri_id
+	 * @return string
+	 */
+	public function addShareLink(string $body, int $quote_uri_id): string
+	{
+		$post = Post::selectFirstPost(['uri', 'plink'], ['uri-id' => $quote_uri_id]);
+		if (empty($post)) {
+			return $body;
+		}
+
+		$link = $post['plink'] ?: $post['uri'];
+		$body .= "\n♲ " . $link;
```

Comment:
Energy is expensive, let's save the creation of an intermediary variable.
```suggestion
		$body .= "\n♲ " . ($post['plink'] ?: $post['uri']);
```